plugin/backend/ipvariant.d

```diff
@@ -647,7 +654,27 @@ body {
         foreach (a; ns.funcRange) {
             generateFuncImpl(a, inner.impl);
         }
-        foreach (a; ns.namespaceRange) {
+
+        foreach (a; ns.classRange) {
+            //a.dumpMethods(10, "jisdf");
+            foreach (b; a.methodPublicRange) { 
+                () @trusted {
```

Comment:
tip: minimize the usage of trusted.
Try to keep as much as possible in the __safe__ subset of D. Memory safe, statically verified at compile time.
[SafeD](http://dlang.org/spec/memory-safe-d.html)
it is probably just b.peek that need it so could be rewritten as
```d
auto cppm = () @trusted { return b.peek!(CppMethod); }();
```